packages/types/lib/account/db.ts

```diff
@@ -1,10 +0,0 @@
-export interface Account {
-    id: number;
-    name: string;
-    secret_key: string;
-    host?: string | null;
-    websockets_path?: string;
-    uuid: string;
-    is_admin?: boolean;
-    is_capped?: boolean;
-}
```

Comment:
The type was the most incorrect, non-existing fields and defined inside types that are user related